src/Function.cpp

```diff
@@ -77,6 +77,21 @@ static long cur_func_idx;				// Index into FuncList that we are currently workin
 static bool param_first=true;			// Flag to track output of commas
 static int builtin_functions_cnt;
 
+void
+BooleanFunctionAttribute::generate_func_attr(int attr_prob, string func_attr)
+{
+	FuncAttrProb = attr_prob;
```

Comment:
I have not used probability yet.

Reply:
Use it, save it in class instance and use it in `output_attr`.